paper.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: page
+title: Paper and Analysis
+---
+
+# Publications and analysis using CSP.guru
+
+csp. guru has beed developed since 2016 by Johan Lilliestam and his lab. The following publications use the data. If you want to cloaborate on additional publications or have used csp.guru for another publication not listed here, please [get in touch](mailto:richard.thonig@iass-potsdam.de).
+
+## Peer-reviewed publications that produced a new version of CSP.guru and explain the data collection procedure.
```

Comment:
This is a very long headline that looks slightly odd in the rendered website. I'd shorten it. Maybe to "Peer-reviewed publications explaining data collection"

Reply:
My problem is, is that I don't "see what I get" locally.